web/readme.md

```diff
@@ -0,0 +1,68 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+### Install npm and node in your computer
+
+npm is just like maven, a package manager for the JavaScript. node is similar to javaSE, JavaScript runtime environment that executes JavaScript code outside of a browser.
+
+#### MacOS
+
+you can use
+```
+brew install node
+brew install npm
+```
+to install them or use MacOS installer, download in https://nodejs.org/en/download/.
+
+#### Windows
+
+I suggest using Windows Installer, download in https://nodejs.org/en/download/.
```

Comment:
I suggest ... -> You can use Windows Installer, which can be downloaded in https://nodejs.org/en/download/.

Reply:
done